lib/modules/versioning/docker/readme.md

```diff
@@ -1,19 +1,31 @@
-Docker doesn't really have _versioning_, instead it supports "tags" and these are usually used by Docker image authors as a form of versioning.
+Docker doesn't really have _versioning_, instead it has "tags".
+Tags are often used by Docker image authors as a form of versioning.
 
-This Docker versioning implementation in Renovate is designed to handle the most common _conventions_ used in tagging images. In particular, it treats the text after the first hyphen as a type of platform/compatibility indicator.
+Renovate tries to follow the most common _conventions_ that are used to tag Docker images.
+In particular, Renovate treats the text after the first hyphen as a type of platform/compatibility indicator.
 
-For example, many images include images with the "-alpine" suffix, e.g. the official `node` Docker image includes tags like `12.15.0-alpine` which is _not_ compatible with `12.15.0` or `12.15.0-stretch`. This means users only want/expect upgrades to `12.16.0-alpine` and not `12.16.0` or `12.16.0-stretch`.
+For example, many images have releases with the `-alpine` suffix.
+The official `node` Docker image has tags like `12.15.0-alpine` which is _not_ compatible with `12.15.0` or `12.15.0-stretch`.
+Users on `-alpine` don't want updates to `12.16.0` or `12.16.0-stretch`.
+Those users only want upgrades to `12.16.0-alpine` and not `12.16.0` or `12.16.0-stretch`.
 
-Similarly, a user with `12.14` expects to be upgraded to `12.15` and not `12.15.0`.
+Similarly, a user on `12.14` expects to be upgraded to `12.15` and not `12.15.0`.
 
 **What type of versioning is used?**
 
-It's pretty "wild west" for tagging and not always compliant with SemVer. Docker versioning in Renovate should do a best effort to accept and sort SemVer-like versions.
+Docker image authors can use whatever tag they want, it's a "wild west".
+Docker tags don't always follow SemVer.
+
+This means that Renovate can only try to accept and sort SemVer-like versions, this won't always work though.
+You may need to help Renovate and create your own custom rules for some Docker images.
+TODO: find link to section that explains creating own Docker versioning/rules.
```

Comment:
I need some help to find the best place to link to. I guess people sometimes need to create their own `packageRules` to work around a badly versioned Docker image tag?

```json
{
  "packageRules": [
    {
      "matchDatasources": ["docker"],
      "matchPackageNames": ["badly-versioned-docker-image"],
      "versioning": "insert-the-correct-string-here"
    }
  ]
}
```

Reply:
Use `loose` versioning in your example